arches/settings_utils.py

```diff
@@ -226,7 +238,6 @@ def generate_frontend_configuration():
         tsconfig_path = os.path.realpath(
             os.path.join(base_path, "..", ".tsconfig-paths.json")
         )
-        sys.stdout.write(f"Writing tsconfig path data to: {tsconfig_path} \n")
```

Comment:
For non-editable installs, this seems to be overwriting in the same location, e.g. at the top-level of site-packages (outside of any namespace). We might want to namespace it with arches, or shove it under arches.

But more importantly, this suggests to me we might get away with just copying the final alias file generated (as it already was) by the app being manage.py'd to a single location (under core)? And then not have to worry about the settings mechanics details.

```sh
Writing tsconfig path data to: /Users/jwalls/release/lib/python3.12/site-packages/.tsconfig-paths.json 
```